Packages/Tracking/Core/Editor/Scripts/LeapServiceProviderEditor.cs

```diff
@@ -125,7 +125,16 @@ private void multiDeviceToggleWithVersionCheck(SerializedProperty property)
 
         private void drawSerialNumberToggle(SerializedProperty property)
         {
-            if (LeapController != null)
+            bool createdController = false;
+            if(_leapController == null)
+            {
+                if(LeapController != null)
```

Comment:
This could do with a comment to state that the act of accessing the controller property will force the assignment of the backing field (if the controller is available on the service provider).